content/en/docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -161,28 +161,38 @@ Kubernetes officially supports client libraries for [Go](#go-client) and
 
 #### Go client
 
-* To get the library, run the following command: `go get k8s.io/client-go/<version number>/kubernetes` See [https://github.com/kubernetes/client-go](https://github.com/kubernetes/client-go) to see which versions are supported.
-* Write an application atop of the client-go clients. Note that client-go defines its own API objects, so if needed, please import API definitions from client-go rather than from the main repository, e.g., `import "k8s.io/client-go/1.4/pkg/api/v1"` is correct.
+* To get the library, run the following command: `go get k8s.io/client-go@kubernetes-<kubernetes-version-number>` See [https://github.com/kubernetes/client-go](https://github.com/kubernetes/client-go) to see which versions are supported.
```

Comment:
I'd like to know if this is always expected to work:
```suggestion
* To get the library, run the following command: `go get k8s.io/client-go@kubernetes-{{< param "fullversion" >}}`. See [https://github.com/kubernetes/client-go](https://github.com/kubernetes/client-go) to see which versions are supported.
```

Reply:
What is `param "fullversion"`?
I try to execute `go get k8s.io/client-go@kubernetes-1.16.1`, it is OK.

Reply:
`{{< param "fullversion" >}}` is a [Hugo shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/) that expands to the version of Kubernetes that the documentation is built against.

Reply:
I found the shortcode in the `tasks/tools/install-kubectl.md` and it was rendered to `v1.16.0`.
I test the version number. It does not work.

Reply:
OK, that makes sense. Thanks for explaining, @diguage.